src/Apps/Conversation/Components/DetailsHeader.tsx

```diff
@@ -42,12 +41,10 @@ export const DetailsHeader: FC<DetailsHeaderProps> = props => {
           <Text variant="sm-display" ml={2}>
             Details
           </Text>
-          <CloseIcon
-            mr={1}
-            mt={0.5}
-            cursor="pointer"
-            onClick={() => setShowDetails(false)}
```

Comment:
This really should be obvious but I encountered this twice: do not put click handlers on anything but buttons (or in some cases, links).